compiler/damlc/daml-compiler/src/DA/Daml/Compiler/Upgrade.hs

```diff
@@ -119,24 +119,48 @@ generateSrcPkgFromLf thisPkgId pkgMap pkg = do
             toNormalizedFilePath $
             (joinPath $ map T.unpack $ LF.unModuleName $ LF.moduleName mod) <.>
             ".daml"
-    pure ( fp
-         , unlines header ++
-           (showSDocForUser fakeDynFlags alwaysQualify $
-            ppr $ generateSrcFromLf thisPkgId pkgMap mod))
+    pure
+        ( fp
+        , unlines header ++
+          (showSDocForUser fakeDynFlags alwaysQualify $
+           ppr $ generateSrcFromLf False thisPkgId pkgMap mod) ++
```

Comment:
Why can’t we use qualified imports when generating the source for a package?

Reply:
Right now it works with unqualified imports for the module we're generating instances for. But we might want to change this in the future.